lib/crawlfs.js

```diff
@@ -1,21 +1,37 @@
 'use strict'
-const fs = process.versions.electron ? require('original-fs') : require('fs')
-const glob = require('glob')
 
-module.exports = function (dir, options, callback) {
-  const metadata = {}
-  return glob(dir, options, function (error, filenames) {
-    if (error) { return callback(error) }
-    for (const filename of filenames) {
-      const stat = fs.lstatSync(filename)
+const pify = require('pify')
+
+const fs = pify(process.versions.electron ? require('original-fs') : require('fs'))
+const glob = pify(require('glob'))
```

Comment:
Not using `glob-promise` because I was trying to minimize the dependencies I added, and `glob-promise` doesn't add much for what we're doing.